org.eclipse.winery.workflowmodeler/webpack.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./config/webpack.dev.js');
```

Comment:
Please migrate to webpack 2 if possible - see https://webpack.js.org/guides/migrating for detailed steps.